lib/puppet/version.rb

```diff
@@ -73,19 +67,19 @@ def self.version
     @puppet_version ||= PUPPETVERSION
   end
 
-  # @return [SemanticPuppet::Version] the puppet version
-  def self.parsed_version
-    @parsed_version ||= SemanticPuppet::Version.parse(version)
+  # @return [String] containing the puppet major version, e.g. "3"
+  def self.version_major
+    self.version.split('.').first
   end
 
-  # @return [String] the minor version as a  "major.minor" string
-  def self.minor_version
-    v = parsed_version
-    "#{v.major}.#{v.minor}"
+  # @return [String] containing the puppet minor version, e.g. "0"
+  def self.version_minor
+    self.version.split('.')[1]
   end
 
-  def self.major_version
-    parsed_version.major.to_s
+  # @return [String] containing the puppet version to minor specificity, e.g. "3.0"
+  def self.version_to_minor
```

Comment:
I liked the previous naming better. This name `version_to_minor` suggest that some kind of transformation takes place, from version to minor.